### ISSUE TITLE
Dataclass support not working

### ISSUE THREAD

**Issue by JulianCologne** [astanin/python-tabulate#200]
hi,

i saw that in #125 the support for dataclasses was introduced.
Unfortunately it doesn't work for me (or I'm doing something wrong^^)

Python 3.10.6
Tabulate: 0.8.10

Example
```pyhton
from tabulate import tabulate
from dataclasses import dataclass, asdict, make_dataclass

@dataclass
class Person:        
    name: str
    age: int

persons = [
    Person("John", 20),
    Person("Jane", 21),
]

# OLD STYLE (working)
print(tabulate((asdict(p) for p in persons), headers="keys"))

# NEW DATACLASS SUPPORT (not working)
print(tabulate(persons, headers="keys"))
```

Error: TypeError: object of type 'Person' has no len()

I checked you test code and found that the example is also not working for me:
```python
Person = make_dataclass("Person", ["name", "age", "height"])
ld = [Person("Alice", 23, 169.5), Person("Bob", 27, 175.0)]
result = tabulate(ld, headers="keys")
```

Any ideas? :)

**Comment by JulianCologne** [comment]
Just noticed that dataclass support did not make it into the current version :(

Any idea when an update might come?
Really like the nice syntax with datasclass support! :D
Thanks!

**Comment by astanin** [comment]
@JulianCologne I published v0.9.0 on PyPI. It has dataclass support and some other major changes.
